app/src/main/kotlin/dev/patrickgold/florisboard/app/settings/clipboard/ClipboardScreen.kt

```diff
@@ -71,6 +72,22 @@ fun ClipboardScreen() = FlorisScreen {
                 stepIncrement = 5,
                 enabledIf = { prefs.clipboard.historyEnabled isEqualTo true && prefs.clipboard.cleanUpOld isEqualTo true },
             )
+            SwitchPreference(
+                prefs.clipboard.autoCleanSensitive,
+                title = "Auto clean sensitive items",
+                enabledIf = { prefs.clipboard.historyEnabled isEqualTo true },
+                visibleIf = { AndroidVersion.ATLEAST_API33_T },
+            )
+            DialogSliderPreference(
+                prefs.clipboard.autoCleanSensitiveAfter,
+                title = "Auto clean sensitive items after",
+                valueLabel = { pluralsRes(R.plurals.unit__seconds__written, it, "v" to it) },
+                min = 0,
+                max = 300,
+                stepIncrement = 5,
```

Comment:
```suggestion
                stepIncrement = 10,
```

I feel like with 10 it is easier to set to the desired value and it is still fine-grained enough for custom values